server/src/database/votings_test.go

```diff
@@ -18,8 +18,6 @@ func TestRunnerForVoting(t *testing.T) {
 
 	t.Run("Update=0", testReopenClosedVotingShouldFail)
 	t.Run("Update=1", testReopenAbortedVotingShouldFail)
-	t.Run("Update=2", testChangeAbortedToClosedVotingShouldFail)
-	t.Run("Update=3", testChangeClosedToAbortedVotingShouldFail)
 	t.Run("Update=4", testCloseVoting)
 	t.Run("Update=5", testCloseVotingUpdateRank)
```

Comment:
Please change `Update=4` and `Update=5` to be 2 and 3 (since the two tests in front of them have been removed)